src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -23,13 +23,16 @@ public ActivityAttribute ()
 #endif
 		public bool                   AllowTaskReparenting    {get; set;}
 		public bool                   AlwaysRetainTaskState   {get; set;}
+		public bool                   AutoRemoveFromRecents   {get; set;}
+		public string                 Banner                  {get; set;}
 		public bool                   ClearTaskOnLaunch       {get; set;}
 #if ANDROID_26
 		public string                 ColorMode               {get; set;}
 #endif
 		public ConfigChanges          ConfigurationChanges    {get; set;}
 #if ANDROID_24
 		public bool                   DirectBootAware         {get; set;}
+		public string                 EnableVrMode            {get; set;}
```

Comment:
This was added in API-24, and thus the declaration should be wrapped in a `#if ANDROID_24` block.